pkg/apis/autoscaling/v1alpha1/metricsproducer.go

```diff
@@ -46,17 +46,38 @@ type PendingCapacitySpec struct {
 	NodeSelector map[string]string `json:"nodeSelector"`
 }
 
-type ScheduledCapacitySpec struct {
-	// NodeSelector specifies a node group. The selector must uniquely identify a set of nodes.
-	NodeSelector map[string]string `json:"nodeSelector"`
+type Timezone string
+
+type ScheduleSpec struct {
 	// Behaviors may be layered to achieve complex scheduling autoscaling logic
 	Behaviors []ScheduledBehavior `json:"behaviors"`
+	// Defaults to UTC. Users will specify their schedules assuming this is their timezone
```

Comment:
Should default to the cluster's local timezone.

Reply:
Are you sure you're able to set a timezone for a cluster?